build.groovy

```diff
@@ -194,6 +194,8 @@ options:
 	cli.cc(longOpt:'ccczUnit', 'Flag to indicate to collect code coverage reports during zUnit step')
 	cli.cch(longOpt:'cccHost', args:1, argName:'cccHost', 'Headless Code Coverage Collector host (if not specified IDz will be used for reporting)')
 	cli.ccp(longOpt:'cccPort', args:1, argName:'cccPort', 'Headless Code Coverage Collector port (if not specified IDz will be used for reporting)')
+	cli.cco(longOpt:'cccOptions', args:1, argName:'cccOptions', 'Headless Code Coverage Collector Options')
```

Comment:
@dennis-behm Actually, the host and port are pretty static, but options can change for each build. It would make sense to have host and port in build-conf only, while the CC options are in the command line or in ZunitConfig.properties.
I just extended the previous implementation, which had the host and the port as parameters of the command line. If agreed, I can change it to have it in property files.

Reply:
Ok, I thought the Headless Code Coverage Collector Options are pretty static. What would be a typical value of it?